src/features/auth/hooks.ts

```diff
@@ -34,37 +34,65 @@ export const useTokenCookie = (
   const dispatch = useAppDispatch();
 
   // Pull token from main cookie. If it exists, and differs from state, try it for auth.
-  const [cookieToken, setCookieToken, clearCookieToken] = useCookie(cookieName);
-  const { isSuccess, isFetching } = useTryAuthFromToken(cookieToken);
+  const [cookieToken, setCookieToken, clearCookieToken] = useCookie(
+    cookieName,
+    process.env.NODE_ENV === 'development'
+      ? {}
+      : { domain: `.${process.env.REACT_APP_KBASE_DOMAIN}` }
+  );
+
+  const { isSuccess, isFetching, isUninitialized } =
+    useTryAuthFromToken(cookieToken);
 
   // Controls for backupCookie
   const [backupCookieToken, setBackupCookieToken, clearBackupCookieToken] =
-    useCookie(backupCookieName);
+    useCookie(backupCookieName, { domain: backupCookieDomain });
 
   // Pull token, expiration, and init info from auth state
   const token = useAppSelector(authToken);
   const expires = useAppSelector(({ auth }) => auth.tokenInfo?.expires);
-  const initialized = useAppSelector(authInitialized);
+  const appAuthInitialized = useAppSelector(authInitialized);
 
   // Initializes auth for states where useTryAuthFromToken does not set auth
   useEffect(() => {
-    if (isFetching || initialized) return;
+    // If the cookieToken is present but it failed checks and wont be overwritten by a token in state, clear
```

Comment:
this is the new condition to clear the cookieToken immediately after the token check call, which always happens before the legacy iframe is loaded (instead of waiting for the iframe to postMessage its login state). This is safe, as the cookie now uses the same domain for Legacy and Europa.